docs/proposals/376-metric-aggregagor/README.md

```diff
@@ -174,25 +174,40 @@ The overall flow looks like:
 
 Let's break down the flow into several steps:
 
-- Step 1: we'll collect the metrics from the inference workloads, we choose `PUSH` mode here just to put less pressure on the gateway side, or the gateway will have iterate all the Pods which obviously will lead to performance issues.
-- Step 2: the gateway plugin will parse the metrics and store them in the redis, this is for HA consideration and cache sharing. Once the instance is down, we can still retrieve the metrics from redis. And if we have multiple instances, we can share the metrics with each other via redis. Considering Envoy AI gateway already uses Redis for limit rating, we'll reuse the Redis here.
-- Step 3 & 4: Traffic comes, and the Router will retrieve the metrics from Redis and make routing decisions based on different algorithms, like queue size aware scheduling.
+- Step 1: we'll collect the metrics from the inference workloads in metrics aggregator.
```

Comment:
Don't use 'PUSH' mode to collect metrics? I think `Push` mode is better.

Reply:
Pull is more simple for small clusters, but we may employ Push in the future see ##Alternative.